packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -175,6 +181,32 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }, [articleData])
 
+  const [borderColor, setBorderColor] = useState('')
```

Comment:
You can set the type of a state. Since you only want values of `TagColor` enum you can write is `.... = useState<TagColor>()`. If you do that you're not able to use `''` as the default value. But you could add another entry to the enum `white` which can be used as a default color.

Reply:
I also suggest to rename `borderColor` to state color. You're using the color for multiple things not just the border.